torch/csrc/jit/codegen/cuda/executor_launch_params.h

```diff
@@ -0,0 +1,91 @@
+
+#include <torch/csrc/jit/codegen/cuda/type.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+
+class TORCH_CUDA_API LaunchParams {
+ public:
+  unsigned int smem() const {
+    return smem_;
+  }
+  unsigned int nBlocks() const {
+    return gdimx_ * gdimy_ * gdimz_;
+  }
+
+  unsigned int nThreads() const {
+    return bdimx_ * bdimy_ * bdimz_;
+  }
+
+  unsigned int bdimx() const {
+    return (unsigned int)bdimx_ == -1 ? 1 : bdimx_;
```

Comment:
why the cast? did you mean 
`(unsigned int)(bdimx_ == -1 ? 1 : bdimx_)` ?
or even better
`static_cast<unsigned int>(bdimx_ == -1 ? 1 : bdimx_)`

Reply:
All syntax options here work, changed to static_cast.